extensions/search-composer-packagist/package.json

```
@@ -0,0 +1,38 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "search-composer-packagist",
+  "title": "Composer Composer Packagist",
+  "description": "Search Composer Packagist For PHP Packages",
+  "icon": "composer-icon.png",
+  "author": "danyelkeddah",
+  "license": "MIT",
+  "contributors": [],
+  "commands": [
+    {
+      "name": "composer",
+      "title": "Search Composer Packagist",
+      "subtitle": "Composer Pacjagist",
```

Here is just a little typo.

```suggestion
      "subtitle": "Composer Packagist",
```